src/main/java/se/sundsvall/smloader/api/JobsResource.java

```diff
@@ -90,4 +90,20 @@ ResponseEntity<Void> dbCleaner(
 			.header(CONTENT_TYPE, ALL_VALUE)
 			.build();
 	}
+
+	@PostMapping(path = "/refreshlabels")
```

Comment:
This method isn't very RESTful.

SOLUTION 1:
`@PostMapping(path = "/labels/refresh")` 

SOLUTION 2 (even more pure):
```
@PatchMapping(path = "/labels", consumes = APPLICATION_JSON_VALUE)
public ResponseEntity<Void> refreshLabels(@RequestBody LabelOperation operation) {
	if ("refresh".equalsIgnoreCase(operation.getOperation())) {
		asyncExecutorService.refreshLabels();
		return noContent().build();
	}
	return badRequest().build();
}

Request:
{ "operation": "refresh" }
```